src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,47 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+export default function Footer() {
+  return (
+    <footer className="fixed bottom-0 left-0 right-0 z-50 bg-gray-100 dark:bg-neutral-900 shadow-inner">
+      <div className="flex items-center justify-between gap-4 px-6 h-20">
+        {/* Developers */}
+        <div className="flex flex-col">
+          <span className="text-xs uppercase tracking-wide text-gray-400 dark:text-gray-500">
+            Developers
+          </span>
+          <div className="flex space-x-6 text-sm text-gray-600 dark:text-gray-300">
```

Comment:
It could be an array with data, that is used to iterate over it with `Link` component